functional/iak-idevid-register-with-certificates/main.fmf

```diff
@@ -0,0 +1,21 @@
+summary:  Test IAK ad IDevID regeneration, registration and cert checking
+description: |
+    Generate iak and idevid certs
+    Check that the agent creates the same keys as tpm2_createprimary
+    Register using the certs, verifying AK with IAK
+contact: Isaac Matthews <isaac@hpe.com>
+component:
+  - keylime
+test: ./test.sh
+framework: beakerlib
+require:
+  - yum
+  - tpm2-tools 
+  - xxd
+recommend:
+  - keylime
+duration: 5m
+enabled: true
+adjust:
+  - when: distro != fedora-39
```

Comment:
Could you please use `when: distro < fedora-39 or distro = centos-stream-9` so we won't have to update it going forward? I believe the test scenario should eventually work for Rawhide too.

Reply:
Resolved by [this commit](https://github.com/RedHat-SP-Security/keylime-tests/pull/514/commits/249742dcf209ed42678d7c8747dda6d949006257)